source/mainnet/net/nodes/baker-docker.rst

```diff
@@ -36,24 +32,45 @@ and send a ``baker add`` transaction to the network:
 where you replace
 
 - ``<amount-to-stake>`` with the CCD amount for the baker's initial stake
-- ``<concordium-data-dir>`` with:
-
-  * on Linux and MacOS: ``~/.local/share/concordium``
-  * on Windows: ``%LOCALAPPDATA%\\concordium``.
-
-  Remember to configure your node to bake using this path.
+- ``<concordium-data-dir>`` with the directory specified as the volume mount for
+  the node database. In the :ref:`sample configuration file<run-a-node>` this is
+  ``/var/lib/concordium-testnet`` for the testnet node, and
+  ``/var/lib/concordium-mainnet`` for the mainnet node.
 
 .. Warning::
    Do not stake all of your funds or you will not have enough funds to cover transaction fees.
 
 Provide a ``--no-restake`` flag to avoid automatically adding the
 rewards to the staked amount on the baker. Read more about this behavior in the section :ref:`Restake earnings<restake-earnings>`.
 
-To start the node with these baker keys and bake blocks, you
-first need to shut down the current running node. To do this, either press ``Ctrl + C`` on the terminal where the node is running or use the
-``concordium-node-stop`` executable.
+To start the node with these baker keys and bake blocks do the following.
+
+1. Stop the currently running node. To do this, press ``Ctrl + C`` on the terminal where the node is running or run
+
+   .. code-block:: console
+
+      $docker stop testnet-node
+
+   or
+
+   .. code-block:: console
+
+      $docker stop mainnet-node
+
+   depending on the environment.
+
+2. Update the configuration file for the node by adding
+
+   .. code-block:: yaml
+
+      - CONCORDIUM_NODE_BAKER_CREDENTIALS_FILE=/mnt/data/baker-credentials.json
+
+   into the ``environment`` section of the ``node`` service section of the file.
 
-When you've placed the file in the appropriate directory, which is what you did you did in the previous command when you specified the output file, start the node again using ``concordium-node``. The node will automatically start baking when the baker is included in the bakers for the current epoch.
+When you've placed the file in the appropriate directory, which is what you did
+in the previous command when you specified the output file, start the node again
+using ``concordium-node``. The node will automatically start baking when the
```

Comment:
I believe this is now wrong? And one should use e.g. docker compose?

Reply:
Yes.